modules/ROOT/pages/event-tracking.adoc

```diff
@@ -17,24 +16,15 @@ Event tracking enables you to view business transactions and events on your Mule
 == Notification Types
 
 Mule runtime engine notifies you about flow component execution.
+See xref:mule-runtime::about-mule-event.adoc[Mule Events].
 
-image::event-tracking.png[event_tracking]
+For the types of notification that Mule provides, see 
```

Comment:
The number used for type or types has to be reflected by the object or objects as well:
```suggestion
For the types of notifications that Mule provides, see 
```